jekyll/_docs/configuration.md

```diff
@@ -291,6 +291,9 @@ You can enable other databases and services from the `services` section:
 ```
 machine:
   services:
+    - mysql
+    - mongodb
+    - postgresql
```

Comment:
this seems to be in conflict with the paragraph above that says they are running by default. doesn't putting them here try to start them?